src/database_async.cc

```diff
@@ -144,18 +135,18 @@ void ReadWorker::HandleOKCallback () {
       Nan::Null()
     , returnValue
   };
-  callback->Call(2, argv);
+  callback->Call(2, argv, async_resource);
 }
 
 /** DELETE WORKER **/
 
-DeleteWorker::DeleteWorker (
-    Database *database
-  , Nan::Callback *callback
-  , leveldb::Slice key
-  , bool sync
-  , v8::Local<v8::Object> &keyHandle
-) : IOWorker(database, callback, key, keyHandle)
+DeleteWorker::DeleteWorker(Database *database,
+                           Nan::Callback *callback,
+                           leveldb::Slice key,
+                           bool sync,
+                           v8::Local<v8::Object> &keyHandle,
+                           const char *resource_name)
```

Comment:
`DeleteWorker` is used both directly and also inherited by `WriteWorker`. Hence putting the `resource_name` parameter last, which will default to `"leveldown:db.del"` in the header file.